actions/yaml_dummy_update_file_commit_push/action.yml

```diff
@@ -46,6 +46,12 @@ inputs:
       Prefix must be unchanged otherwise old lines cannot be found
     required: false
     default: 'true'
+  commit-user-name:
+    description: User to be used as committer
```

Comment:
"...as Git committer"
(prefer to add Git for clarity, same for other places)

Reply:
done